storage/ctab/src/main/java/org/openscience/cdk/io/SDFWriter.java

```diff
@@ -273,7 +273,6 @@ private void writeMolecule(IAtomContainer container) throws CDKException {
             mdlWriter.addSettings(getSettings());
             mdlWriter.write(container);
             mdlWriter.close();
-            writer.write(stringWriter.toString());
```

Comment:
@johnmay, you may want to check out the changes in this file.

Reply:
They look fine, I had a similar patching waiting to find time to apply